src/client/components/ChangesetList/ChangesetList.js

```diff
@@ -77,29 +86,40 @@ class ChangesetList extends Component {
     this.setState({ activeKey })
   }
 
-  handleChange = (obj, isSelected) => {
+  handleChange = (event) => {
+
+    const isSelected = event.target.checked
+    const returnValue = false
+
     if (this.state.changesets.length < 2 && isSelected) {
-      this.state.changesets.push(obj.target.value)
+      this.state.changesets.push(event.target.value)
     }
     if (this.state.changesets.length <= 2 && !isSelected) {
-      const index = this.state.changesets.indexOf(obj.target.value)
+      const index = this.state.changesets.indexOf(event.target.value)
       if (index !== -1) {
         this.state.changesets.splice(index, 1)
       }
       this.props.commits.map((item) => {
-        const ch = this.refs[item.hash]
+        const ref = item.id
+        const ch = this.refs[ref]
         ch.setState({ disabled: false })
-        return true
+        const returnValue = true
```

Comment:
This creates a new variable inside the map's callback scope which means nothing and returns nothing. Remove both lines, I don't see much sense in both :)